tests/integration/adapter/queries-test.js

```diff
@@ -57,6 +57,42 @@ test("When a query is made, the adapter should receive a record array it can pop
   }));
 });
 
+test("a query can be updated via `update()`", function(assert) {
+  let done = assert.async();
```

Comment:
You are absolutely right :+1: Removed the `done` and the amount of expected assertions, since this is handled by the run-loop...